source/extensions/filters/common/local_ratelimit/local_ratelimit_impl.cc

```diff
@@ -48,6 +48,7 @@ LocalRateLimiterImpl::LocalRateLimiterImpl(
       throw EnvoyException(
           "local rate descriptor limit is not a multiple of token bucket fill timer");
     }
+    // Save the multiplicative factor to control the descriptor refill frequency.
```

Comment:
Sorry can you rename token_bucket to per_descriptor_token_bucket or something? I think that was the part that confused me the most. Then the rest makes sense, thank you.

/wait